CONTRIBUTING.md

```diff
@@ -14,11 +14,13 @@ We welcome contributions to `Rgemini` by all GEMINI team members. To submit a co
 
 6. If appropriate, add unit tests in the `tests/testthat/` directory.
 
-7. Submit a [pull request](https://help.github.com/articles/using-pull-requests) into the `develop` branch.
+7. If adding new functions with new documentation, run `devtools::document()` and update the `_pkgdown.yml` file by adding the new function to the appropriate reference section.
```

Comment:
Should we remove the part "run `devtools::document()`" since we now have the documentation step automated?